puppet/modules/koji/manifests/mirror.pp

```diff
@@ -0,0 +1,48 @@
+# Basic mirror for Koji
+class koji::mirror {
+  ensure_packages(['dnf-plugins-core'])
+
+  file { '/srv/mirror':
```

Comment:
I'd also make this a class parameter, given the repetition.

Reply:
ack